src/main/java/ladder/LadderMain.java

```diff
@@ -1,12 +1,30 @@
 package ladder;
 
 import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
 
 public class LadderMain {
     public static void main(String[] args) {
         List<String> names = InputView.inputPersonNames();
+        List<String> resultList = InputView.inputPlayResults();
         int ladderHeight = InputView.inputLadderHeight();
         Ladder ladder = new Ladder(ladderHeight, names.size(), new HorizontalLineRandomStrategy());
-        OutputView.printLadder(names, ladderHeight, ladder);
+        OutputView.printLadder(names, ladderHeight, ladder, resultList);
+
+        Persons persons = new Persons(names);
+        Results results = new Results(resultList);
+        String name = InputView.inputResultName();
+        if (name.equals("all")) {
```

Comment:
`LadderMain`은 View 영역과 도메인 영역을 연계해주는 컨트롤러 역할이라고 생각되는데요. 컨트롤러에 로직을 구성하기 보다 다른 영역에서 처리함은 어떠실까요?

Reply:
```suggestion
        if ("all".equals(name)) {
```
추가로 name이 null이라면 NullPointerException이 발생할텐데요. String인게 명확한 `all`을 사용해서 equals 호출하면 NullPointerException을 방지할 수 있을꺼에요.